scripts/packaging/linux_distro.py

```diff
@@ -67,6 +57,13 @@ class LinuxWebotsPackage(WebotsPackage):
         "libzip.so.4",  # needed by Robotis OP2
         "libx264.so.163"
     ]
+    USR_LIB_X68_64_24_04 = [
+        "libIex-3_1.so.30",
+        "libIlmThread-3_1.so.30",
+        "libwebp.so.7",
```

Comment:
Here we are missing some libraries. We discussed them, but just leaving a note

Reply:
I believe these libraries are related to the tar distribution. It would be good to test the tar distribution on a clean Ubuntu 24.04